website/www/site/layouts/partials/feedback.html

```diff
@@ -0,0 +1,22 @@
+{{/*
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. See accompanying LICENSE file.
+   */}}
+
+<div class="feedback">
+    {{ if ne (.Page.Lastmod.Format "2006/01/02") "0001/01/01" }}
+        <p class="update">Last updated on {{ .Page.Lastmod.Format "2006/01/02" }}</p>
+    {{ end }}
+    <h3>{{ T "feedback-title" }}</h3>
+    <p class="description">
+        {{ T "feedback-description" }}
+    </p>
+    <button class="load-button"><a href="mailto:dev@beam.apache.org">{{ T "feedback-button" }}</a></button>
```

Comment:
Let's put a default subject on this. I believe this is the right syntax:
```suggestion
    <button class="load-button"><a href="mailto:dev@beam.apache.org?subject=Beam Website Feedback">{{ T "feedback-button" }}</a></button>
```